MediaAdminBundle/ExtractReference/ExtractReferenceManager.php

```diff
@@ -59,4 +59,29 @@ public function getReferencePattern(StatusableInterface $statusableElement)
 
         throw new ExtractReferenceStrategyNotFound();
     }
+
+    /**
+     * Get invalidate tag of references
+     *
+     * @param array $references
+     *
+     * @return array
+     */
+    public function getInvalidateTagStatusableElement(array $references)
+    {
+        $invalidateTag = array();
+        /** @var ExtractReferenceInterface $strategy */
+        foreach ($references as $reference) {
+            foreach ($this->strategies as $strategy) {
+                if ($strategy->supportReference($reference)) {
+                    $tag = $strategy->getInvalidateTagStatusableElement($reference);
+                    if (null !== $tag) {
+                        $invalidateTag[] = $tag;
+                    }
+                }
+            }
+        }
+
+        return $invalidateTag;
```

Comment:
@alavieille : an array is return but the name of the method and the phpdoc on the interface both say only a string should be returned